_includes/releases/babelfish-1.2.4.md

```diff
@@ -0,0 +1,23 @@
+- Babelfish Version: 1.2.4
+- Extension Version: 1.2.1
+- PostgreSQL Server Version: 13.6
+- Download source distributions:
+  - [BABEL_1_2_4__PG_13_6.zip](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_1_2_4__PG_13_6/BABEL_1_2_4__PG_13_6.zip)
+  - [BABEL_1_2_4__PG_13_6.tar.gz](https://github.com/babelfish-for-postgresql/babelfish-for-postgresql/releases/download/BABEL_1_2_4__PG_13_6/BABEL_1_2_4__PG_13_6.tar.gz)
+- Babelfish Compass
+  - [Download](https://github.com/babelfish-for-postgresql/babelfish_compass/releases)
+- Date: December 20, 2022
+
+## Overview
+
+Babelfish 1.2.4 contains the following fix that improves the user experience:
+
+- Fixed an issue that prevents a successful minor version upgrade from Babelfish PostgreSQL 13.4 to PostgreSQL 13.5.
```

Comment:
This should be minor version upgrade from 13.4 to 13.6.

Reply:
Updated...